components/link/index.css

```diff
@@ -36,29 +43,45 @@
 	outline: none;
 	cursor: pointer;
 
-	/* @deprecated --mod-link-text-color-primary-default in favor of --mod-link-text-color */
-	color: var(--highcontrast-link-text-color, var(--mod-link-text-color, var(--mod-link-text-color-primary-default, var(--spectrum-accent-content-color-default))));
+	color: var(--highcontrast-link-text-color, var(--mod-link-text-color, var(--spectrum-accent-content-color-default)));
+	font-family: var(--spectrum-link-default-font-family);
+	font-weight: var(--spectrum-link-font-weight);
+	font-size: var(--spectrum-link-font-size);
+	font-style: var(--spectrum-link-font-style);
+	line-height: var(--spectrum-link-line-height);
+	text-decoration-thickness: var(--spectrum-link-text-underline-thickness);
+	text-underline-offset: var(--spectrum-link-text-underline-gap);
 
 	&:hover {
-		/* @deprecated --mod-link-text-color-primary-hover in favor of --mod-link-text-color-hover */
-		--mod-link-text-color: var(--mod-link-text-color-hover, var(--mod-link-text-color-primary-hover, var(--spectrum-accent-content-color-hover)));
+		--mod-link-text-color: var(--mod-link-text-color-hover, var(--spectrum-accent-content-color-hover));
```

Comment:
This is really cool and a nice efficiency! Love the consistency you implemented it with throughout as well.